### PR TITLE
Apply General fixes to the lesson

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**

### CODE REVIEW

File: _episodes/11-skim-milk.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ skim_ggH:
 
 Ok, fine. That was way too easy. It seems we have a few issues to deal with.
 
-1. The built code in the `build_skim` job (of the `build` stage) isn't in the `skim_ggH` job by default. We need to use GitLab `artifacts` to copy over this from the right job (and not from `build_skim_latest`).
+1. The code in the `multi_build` jobs (of the `build` stage) isn't in the `skim_ggH` job by default. We need to use GitLab `artifacts` to copy over this from the one of the jobs (let's choose as an example the `multi_build: [rootproject/root:6.28.10-ubuntu22.04]` job).
```
Comment (goi42) [inline _episodes/11-skim-milk.md:82]: `build_skim` is still present in "Set The Right Image" below

File: _episodes/11-skim-milk.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ Since the build artifacts don't need to exist for more than a day, let's add art
 
 > ## Adding Artifacts
 >
-> Let's add `artifacts` to our jobs to save the `build/` directory. We'll also make sure the `skim_ggH` job has the right `dependencies` as well. In this case the job multi_build is actually running two parallel jobs: one for the root version 6.28 and the other for the latest version of root. So we have to make sure we specify the right dependency as "multi_build: [rootproject/root:6.28.10-ubuntu22.04]".
+> Let's add `artifacts` to our jobs to save the `skim` binary. We'll also make sure the `skim_ggH` job has the right `dependencies` as well. In this case the job multi_build is actually running two parallel jobs: one for the ROOT version 6.28 and the other for the latest version of ROOT. So we have to make sure we specify the right dependency as `"multi_build: [rootproject/root:6.28.10-ubuntu22.04]"`.
```
Comment (goi42) [inline _episodes/11-skim-milk.md]:
```suggestion
> Let's add `artifacts` to our jobs to save the `skim` binary. We'll also make sure the `skim_ggH` job has the right `dependencies` as well. In this case the job `multi_build` is actually running two parallel jobs: one for the ROOT version 6.28 and the other for the latest version of ROOT. So we have to make sure we specify the right dependency as `"multi_build: [rootproject/root:6.28.10-ubuntu22.04]"`.
```

File: _episodes/11-skim-milk.md [hunk 3]
```diff
@@ -234,7 +234,20 @@ script:
 script:
   - ./skim root://eospublic.cern.ch//eos/root-eos/HiggsTauTauReduced/GluGluToHToTauTau.root skim_ggH.root 19.6 11467.0 0.1
 ```
-{: .language-yaml}
+## Get the output as an artifact
+Finally, let's retrieve the output as an artifact and let's have it experie in 1 week. (Remember that the output of this script is `skim_ggH.root`)
```
Comment (goi42) [inline _episodes/11-skim-milk.md]:
```suggestion
Finally, let's retrieve the output as an artifact and have it expire in 1 week. (Remember that the output of this script is `skim_ggH.root`)
```

File: _episodes/11-skim-milk.md (hunk 1 above)
Comment (GuillermoFidalgo) [reply _episodes/11-skim-milk.md:82]: I see, thanks! Will update it

**Pull request merged**